apm-collector/apm-collector-configuration/collector-configuration-provider/src/test/resources/component-libraries.yml

```diff
@@ -124,6 +124,9 @@ Hystrix:
 Jedis:
   id: 30
   languages: Java
+SOFARPC:
+  id: 31
```

Comment:
the id isn't equals `ComponentsDefine` define's

Reply:
@ascrutae I found  that test  component-libraries.yml 

Jedis has used 30 ,cloud I use 30 too?

Reply:
No, the id must unique

Reply:
@ascrutae It is test file ,seems no one use It, I will remove this line

Reply:
Sorry，I mean the value that it define in `component-libraries.yml`file isn't equals the value define in `ComponentsDefine.java` file